src/myrtlespeech/data/preprocess.py

```diff
@@ -222,3 +225,75 @@ def __repr__(self) -> str:
             + f" n_feature_masks={self.n_feature_masks},"
             + f" n_time_masks={self.n_time_masks})"
         )
+
+
+class MFCCLegacy:
+    """Legacy MFCC implementation using ``python_speech_features``.
+
+    This is required for backwards compatibility: **new users should not use
+    this class.** This class has the same API as
+    :py:class`torchaudio.transforms.MFCC`.
+
```

Comment:
Should this describe what the difference is and what it's being backwards compatible with? (i.e. just add for models trained on mfcc features from python_speech_features?).

It also doesn't have the same API as `:py:class`torchaudio.transforms.MFCC`.`. That API is:

```
torchaudio.transforms.MFCC(sample_rate=16000, n_mfcc=40, dct_type=2, norm='ortho', log_mels=False, melkwargs=None)
```

Reply:
Good point - updated.